.travis.yml

```diff
@@ -10,13 +10,30 @@ python:
 addons:
   postgresql: '9.4'
```

Comment:
Shouldn't this node be the one that's update to the postgresql 13 like in the other repos?

Reply:
You are right. Thanks for catching that.